rosidl_default_runtime/package.xml

```diff
@@ -17,6 +17,8 @@
   <build_export_depend>rosidl_typesupport_introspection_c</build_export_depend>
   <build_export_depend>rosidl_typesupport_introspection_cpp</build_export_depend>
 
+  <group_depend>rosidl_typesupport_c_packages</group_depend>
+  <group_depend>rosidl_typesupport_cpp_packages</group_depend>
   <!-- These two dependencies cannot stay here. -->
   <!-- This is a workaround for: https://github.com/ros2/ros2/issues/174 -->
```

Comment:
if this is closed and resolved we should remove them

Reply:
This is unrelated to this patch. I don't know enough about this to answer the question if the following line can be removed.